### PR TITLE
Writing again issue templates

### PR DESCRIPTION
**Pull request by VaneskaSousa**
Trying to fix some configuration problem with issue templates I decide to write again, directly in totalcross repository

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
+To reproduce the error, the code must also be sent. It can be the origin of your application or an example (as long as it faithfully reproduces the error you are getting in the original application). 
+
+Depending on the case, it is also important to describe the steps to reproduce the error: 
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+## Expected behavior
```
Comment (ItaloYeltsin) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: If person have to write the expected behavior it also must write the Actual behavior.

Comment (ItaloYeltsin) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: Add ## Actual behavior

Comment (VaneskaSousa) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: But the actual behavior is described in ##describe the bug. e.g: ## Describe the bug: MessageBox title doesn't appear; ## Expected behavior: the messagebox title must to appear like in the sdk version 4.4.2. 

I use the [GitLab issue 680](https://gitlab.com/totalcross/TotalCross/-/issues/680) for example

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
+A clear and concise description of what you expected to happen.
+
+## Screenshots or videos
+If applicable, add screenshots or videos to help explain your problem.
```
Comment (ItaloYeltsin) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: add **some** screenshots **and/or** videos to help **to** explain your problem.

Comment (VaneskaSousa) [reply .github/ISSUE_TEMPLATE/bug_report.md]: done 👍

**Pull request merged**